www/js/app.js

```diff
@@ -67,7 +67,32 @@ angular.module('starter', ['ionic', 'starter.controllers'])
         controller: 'PlaylistCtrl'
       }
     }
+  })
+
+  .state('app.home', {
+    url: '/home',
+    views: {
+      'menuContent': {
+        templateUrl: 'templates/home.html'
+      }
+    }
+  })
+
+  .state('app.rlogin', {
+    url: '/rlogin',
+    views: {
+      'menuContent': {
+        templateUrl: 'templates/rlogin.html'
+      }
+    }
   });
   // if none of the above states are matched, use this as the fallback
-  $urlRouterProvider.otherwise('/app/playlists');
+  $urlRouterProvider.otherwise('/app/rlogin');
+});
+
+app.directive("rcLogo", function(){
+  return {
+    require: "E",
+    templateUrl: "templates/logo.html"
+  }
```

Comment:
The routes and directives will end up in their own files later but fine for now.